tests/test_types.py

```diff
@@ -4029,11 +4028,11 @@ class Foobar(BaseModel):
     # insert_assert(exc_info.value.errors(include_url=False))
     assert exc_info.value.errors(include_url=False) == [
         {
-            'type': 'string_too_short',
+            'type': 'too_short',
             'loc': ('password',),
-            'msg': 'String should have at least 6 characters',
+            'msg': 'Value should have at least 6 items after validation, not 0',
             'input': '',
-            'ctx': {'min_length': 6},
+            'ctx': {'field_type': 'Value', 'min_length': 6, 'actual_length': 0},
         }
```

Comment:
Since `SecretStr` is a composite type it will no longer get the `string_too_short` error and instead will get the more general `too_short` error. It seems like the latter is maybe even better, it includes the actual length in the context for starters. I'd be +1 for getting rid of `string_too_short` altogether.